### ISSUE TITLE
Programmatically Reveal (e.g., for didSelectRowAtIndexPath)

### ISSUE THREAD

**Issue by Wirsing84** [alexzielenski/ZKRevealingTableViewCell#14]
Hi there,
thank you for the great implementation!
I'm looking for a way to programmatically reveal the backView when a cell is clicked

What I tried is adding "_slideOutContentViewInDirection" to the .h-File and access it manually like this:
- (void)tableView:(UITableView _)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath{
  ZKRevealingTableViewCell *cell = (ZKRevealingTableViewCell_)[tableView cellForRowAtIndexPath:indexPath];
  
  [cell _slideOutContentViewInDirection:ZKRevealingTableViewCellDirectionLeft];
  }

Basically this works fine, but I'm rather sure that I'm messing things up internally by accessing this private method. What do I need to take care about myself now?

Thanks for your help!

**Comment by alexzielenski** [comment]
If you set the isRevealing property on the cell it will automatically hide or show the backView
